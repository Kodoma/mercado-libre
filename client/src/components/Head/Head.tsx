import React from 'react';
import Head from 'next/head';

const HeadSeo = () => {
    return (
        <Head>
            <title>Codo a codo desde casa en Mercado Libre Argentina</title>
            <meta
                name="description"
                content="Encontrá lo que buscás en Codo a codo desde casa. Todo lo que necesitas lo conseguís en un solo lugar, en Mercado Libre"
                data-head-react="true"
            />
            <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
            <meta
                name="viewport"
                content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
            />
            <meta name="HandheldFriendly" content="True" />
            <meta httpEquiv="cleartype" content="on" />
            <meta name="browser-support" content="samesite=true" />
            <link
                href="https://http2.mlstatic.com/frontend-assets/ui-navigation/5.14.5/mercadolibre/favicon.svg"
                rel="icon"
                data-head-react="true"
            />
            <link
                href="https://http2.mlstatic.com/frontend-assets/ui-navigation/5.14.5/mercadolibre/180x180.png"
                rel="apple-touch-icon"
                data-head-react="true"
            />
            <link
                rel="stylesheet"
                type="text/css"
                charSet="UTF-8"
                href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css"
            />
            <link
                rel="stylesheet"
                type="text/css"
                href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css"
            />
        </Head>
    );
};

export default HeadSeo;
