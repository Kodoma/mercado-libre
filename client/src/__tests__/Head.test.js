import React from 'react';
import 'jsdom-global/register';
import {shallow} from 'enzyme';
import Head from "../components/Head/Head";


describe('__tests__ Head', () => {
    it('should render my Head component', () => {
        const component = shallow(<Head />);
        expect(component.getElements()).toMatchSnapshot();
    });

});