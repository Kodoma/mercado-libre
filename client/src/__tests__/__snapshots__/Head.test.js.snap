// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__tests__ Head should render my Head component 1`] = `
Array [
  <Head>
    <title>
      Codo a codo desde casa en Mercado Libre Argentina
    </title>
    <meta
      content="Encontrá lo que buscás en Codo a codo desde casa. Todo lo que necesitas lo conseguís en un solo lugar, en Mercado Libre"
      data-head-react="true"
      name="description"
    />
    <meta
      content="IE=edge"
      httpEquiv="X-UA-Compatible"
    />
    <meta
      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
      name="viewport"
    />
    <meta
      content="True"
      name="HandheldFriendly"
    />
    <meta
      content="on"
      httpEquiv="cleartype"
    />
    <meta
      content="samesite=true"
      name="browser-support"
    />
    <link
      data-head-react="true"
      href="https://http2.mlstatic.com/frontend-assets/ui-navigation/5.14.5/mercadolibre/favicon.svg"
      rel="icon"
    />
    <link
      data-head-react="true"
      href="https://http2.mlstatic.com/frontend-assets/ui-navigation/5.14.5/mercadolibre/180x180.png"
      rel="apple-touch-icon"
    />
    <link
      charSet="UTF-8"
      href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css"
      rel="stylesheet"
      type="text/css"
    />
    <link
      href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css"
      rel="stylesheet"
      type="text/css"
    />
    <link
      href="/css/Search.module.css"
      rel="stylesheet"
    />
  </Head>,
]
`;
